server/routes/schedules.js

```diff
@@ -169,24 +168,26 @@ const formatSchedule = (workbook) => {
         for (let j = 0; j < temp.length; j++) {//each object in the sheet
           let object = temp[j] //object stores each teacher's schedule
           var keys = Object.keys(object)//get name of keys
-          //for creating classes
-          /*if(object['Course Title'] === undefined){break;}
-          data.push({ "teachable": object['Teachable'], 
-                  "coursecode": object['Course Code'], 
-                  "coursetitle": object['Course Title'],
-                  "grade": object['Grade'],
-                  "pathway": object['Pathway']
-          });
-
-         createClass(data[j]);*/
+          // for creating classes
+        //   if(object['Course Title'] === undefined){break;}
+
+        //   data.push({ "teachable": object['Teachable'], 
+        //           "coursecode": object['Course Code'], 
+        //           "coursetitle": object['Course Title'],
+        //           "grade": object['Grade'],
+        //           "pathway": object['Pathway']
+        //   });
+        //   createClass(data[j]);
+          
            //formatting  schedules:
           if(object['Teacher Name'] === undefined){ break; }
           data.push({ "name": object['Teacher Name'], 
                   "period1": formatPeriod(object['Period 1']), "period1Location": object['__EMPTY'] === undefined? undefined: `${object['__EMPTY']}`, 
                   "period2": formatPeriod(object['Period 2']), "period2Location": object['__EMPTY_1'] === undefined? undefined: `${object['__EMPTY_1']}`,
                   "period3": formatPeriod(object['Period 3']), "period3Location": object['__EMPTY_2'] === undefined? undefined: `${object['__EMPTY_2']}`,
                   "period4": formatPeriod(object['Period 4']), "period4Location": object['__EMPTY_3'] === undefined? undefined: `${object['__EMPTY_3']}`
-          });
+          }); 
+          createTeacher(object['Teacher Name']);
```

Comment:
Is this a line you added or is this a result of the merge wackiness? If this isn't part of your changes we should remove this line

Reply:
i don't remember adding this line. I will remove it.

Reply:
wait which line are you talking about? 189?